packages/milo-http-server-addon/test/http-server.test.js

```
@@ -0,0 +1,23 @@
+import { HttpServer } from '../lib/http-server.js';
+import { test } from 'node:test';
+import assert from 'node:assert';
+import { Response } from '../lib/response.js';
+
+test('HttpServer', async () => {
+    const body = "Hello, World!\r\n";
+    const server = new HttpServer({ hostname: "0.0.0.0", port: 8001 });
+    server.listen((req, err) => {
```

currently errors are swallowed for some reason 😅